### PR TITLE
Update to LFE 2.1.3

### PR DESCRIPTION
**Pull request by BNAndras**
Once exercism/lfe-test-runner#58 is merged, the CI will target LFE 2.1.3 so we can update the rebar configs and locks.

### CODE REVIEW

**Comment by BNAndras** [comment]
Once I have the review approval, I can go ahead and merge the test-runner PR. Once the image is published, I should be able to add an empty commit here to trigger a new CI run, and everything should be good to go (hopefully!).

**Review by kotp — APPROVED** [review]

**Comment by kotp** [comment]
I retriggered.

> Once I have the review approval, I can go ahead and merge the test-runner PR. Once the image is published, I should be able to add an empty commit here to trigger a new CI run, and everything should be good to go (hopefully!).

I have triggered without an empty commit.  Waiting on it now.

**Comment by BNAndras** [comment]
CI fails. The updated test runner is timing out on the website, probably because it's missing the updated files in this PR. So I think we need to merge this as-is and figure out what the problem with phone-number is.

**Pull request merged**
